hazelcast/src/main/java/com/hazelcast/agrona/collections/Int2ObjectHashMap.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2014 Real Logic Ltd.
```

Comment:
Not sure about this... as far as I could study it, the point is to make the attribution in `NOTICE.TXT`. What we have is derivative work.